lib/catalog/martyrology.ts

```diff
@@ -2482,6 +2658,19 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
         or: [679, 680],
       },
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/saint-leon-de-carentan-eveque-et-martyr
+    // - https://fr.wikipedia.org/wiki/L%C3%A9on_de_Carentan
+    leon_de_carentan_bishop: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Leon de Carentan',
```

Comment:
I’d prefer `Leo of Carentan` in English, cf [Wikipedia](https://en.wikipedia.org/wiki/Chronological_list_of_saints_in_the_9th_century#:~:text=Leo%20of%20Carentan).

Remember to update the key and its value in `en.ts` too.

```suggestion
      name: 'Leo of Carentan',
```